src/wasm.h

```diff
@@ -924,6 +924,11 @@ class Module {
   std::vector<UserSection> userSections;
   std::vector<std::string> debugInfoFileNames;
 
+  // `features` are the features allowed to be used in this module and should be
+  // respected regardless of the value of `hasFeaturesSection`.
```

Comment:
maybe also add that `hasFeaturesSection` means that we read one, and will write one (unless told not to)

Reply:
done.